lumen/ai/agents.py

```diff
@@ -740,16 +765,7 @@ async def answer(self, messages: list | str):
         if message.object is None:
             return
         sql_out = message.object.replace("```sql", "").replace("```", "").strip()
-
-        # TODO: find a better way to replace for all tables
-        for table in memory.get("closest_tables", [table]):
-            sql_expr = source.get_sql_expr(table)
-            if f"FROM {table}" in sql_out:
-                sql_in = sql_expr.replace("SELECT * FROM ", "").replace(
-                    "SELECT * from ", ""
-                )
-                sql_out = sql_out.replace(f"FROM {table}", f"FROM {sql_in}")
-            memory["current_sql"] = sql_out
```

Comment:
The template instructs it to use read_parquet if necessary.

Reply:
This is great!